lib/toolbar.html

```diff
@@ -26,7 +38,7 @@
         >
         <input
           type="file"
-          accept=".xlsx,.xls,.tsv,.csv,.json"
+          accept=".json"
```

Comment:
I think you were probably testing this - or maybe reusing?  At any rate we need ".xlsx,.xls,.tsv,.csv," back in there - or else regular data files can't be loaded.  A test data file is in the cancogen template folder under example input. Maybe this means a separate input for data load rather than template load.

Reply:
Change should have been under upload template, but put it under open file